src/components/MediaCollection/createOnBeforeMediaEvent.js

```diff
@@ -0,0 +1,41 @@
+import isBlankString from "../../utils/isBlankString";
```

Comment:
The name of this file is weird. It actually handles the response from edge to start a new session.